### DISCUSSION TITLE
Components for Stimulus

### DISCUSSION THREAD

**Discussion by dennis-g** [tailwindlabs/headlessui#824] [Ideas]
Hello Team!

It would be great, if you could provide your HeadlessUI components for Stimulus (https://stimulus.hotwired.dev/). Have you ever thought about it? It would be very useful to use your components in Symfony with Twig.

Kind regards
Dennis

**Comment by lb-** [comment]
See also
 https://github.com/tailwindlabs/headlessui/discussions/704
https://github.com/tailwindlabs/tailwindcss/discussions/7546
